userpanel/modules/finances/style/bclean/templates/userassignments.html

```diff
@@ -3,20 +3,27 @@
 {$speed_unit_type = ConfigHelper::getConfig('userpanel.speed_unit_type')}
 {$speed_unit_aggregation_threshold = ConfigHelper::getConfig('userpanel.speed_unit_aggregation_threshold')}
 {$count = 0}
+
+<style>
+	.table-thead {
+		color: white;
+	}
+</style>
+
 <div class="table-responsive">
-	<table class="table table-bordered table-striped table-condensed">
-		<tr class="bg-primary text-white">
+	<table class="table table-sm table-hover table-bordered table-striped table-condensed">
+		<thead class="bg-secondary table-thead">
```

Comment:
jeśli to ma mieć znaczenie to może  w style.css:
lms-userpanel-userassignment-head
?
Przecież jak podefiniujesz klasy w stylach w poszczególnych szablonach to potem nie podmienisz tego jednym style.css.

Reply:
Tyczy się to wszystkich takich miejsce gdzieś używasz styli w elemencie `style` (oczywiście nie wszystkich) - czasem drogą wyjątku można użyć elementu `style` w szablonie jak coś ma zastosowanie lokalne. A może tutaj jakiś ogólna klasa `lms-userpanel-table-header`?

Reply:
Wiesz co? Te tabelki z bootstrapa fajnie się mieszają ale wyglądają trochę dziadowo domyślnie.
Zanim zacznę grzebać w stylach wolałbym zrobić research jak to może ładniej wyglądać. Proponuję zostawić jak jest, obecnie wygląda wystarczająco.

Reply:
Ok zaraz zrobię tak

Reply:
Zrobione. Tak ma być?